views/default/walled_garden.css.php

```diff
@@ -2,82 +2,65 @@
 /**
  * Walled garden CSS
  */
-
-$url = elgg_get_site_url();
-
 ?>
 /* <style> /**/
 
-.elgg-body-walledgarden {
-	margin: 100px auto 0 auto;
-	position: relative;
-	width: 530px;
-}
-.elgg-module-walledgarden {
-	position: absolute;
-	top: 0;
-	left: 0;
-}
-.elgg-module-walledgarden > .elgg-head {
-	height: 17px;
-}
-.elgg-module-walledgarden > .elgg-body {
-	padding: 0 10px;
-}
-.elgg-module-walledgarden > .elgg-foot {
-	height: 17px;
-}
-.elgg-walledgarden-double > .elgg-head {
-	background: url(<?= elgg_get_simplecache_url("walled_garden/two_column_top.png"); ?>) no-repeat left top;
-}
-.elgg-walledgarden-double > .elgg-body {
-	background: url(<?= elgg_get_simplecache_url("walled_garden/two_column_middle.png"); ?>) repeat-y left top;
-}
-.elgg-walledgarden-double > .elgg-foot {
-	background: url(<?= elgg_get_simplecache_url("walled_garden/two_column_bottom.png"); ?>) no-repeat left top;
-}
-.elgg-walledgarden-single > .elgg-head {
-	background: url(<?= elgg_get_simplecache_url("walled_garden/one_column_top.png"); ?>) no-repeat left top;
-}
-.elgg-walledgarden-single > .elgg-body {
-	background: url(<?= elgg_get_simplecache_url("walled_garden/one_column_middle.png"); ?>) repeat-y left top;
-}
-.elgg-walledgarden-single > .elgg-foot {
-	background: url(<?= elgg_get_simplecache_url("walled_garden/one_column_bottom.png"); ?>) no-repeat left top;
+body {
+	background-image: url('<?php echo elgg_get_simplecache_url('walled_garden.jpg'); ?>');
```

Comment:
We don't really need to use php for this. 
`background-image: url(walled_garden.jpg);`